auth/build.gradle

```diff
@@ -30,8 +30,8 @@ android {
 dependencies {
     implementation project(path: ':core')
     implementation 'net.openid:appauth'
-    //noinspection GradleCompatible
-    implementation 'com.android.support:customtabs:26.1.0'
+    implementation "com.android.support:customtabs:26.1.0"
```

Comment:
I'm still actually not sure if this change is not dangerous. It simply replaces dependency, but `appauth` is still developed against 26.1.0.
What if we increase it to 27.0.2? 
```
implementation "com.android.support:customtabs:27.0.2"
    implementation "com.android.support:animated-vector-drawable:27.0.2"
    implementation "com.android.support:appcompat-v7:27.0.2"
    implementation('net.openid:appauth', {
        exclude group: 'com.android.support', module: 'customtabs'
    })
implementation('com.scottyab:rootbeer-lib:0.0.6', {
        exclude group: 'com.android.support', module: 'animated-vector-drawable'
        exclude group: 'com.android.support', module: 'appcompat-v7'
    })    
```
So far I tried it and app works still ok. Can we use it?

Reply:
@wojta Might work because we have a hardcoded blacklist against the usage of custom tabs - see [here](https://github.com/aerogear/aerogear-android-sdk/blob/38a514725a31723d09038c7e59952cf1c857cae6/auth/src/main/java/org/aerogear/mobile/auth/authenticator/AuthorizationServiceFactory.java#L86-L88) (which needs to change). Have you tried without the blacklist?

Reply:
@danielpassos Any opinion on using this?

Reply:
If we are not going to use this I see no problem